frontend/src/TransitPredictionsPage.js

```diff
@@ -79,29 +102,29 @@ export default function TransitPredictionsPage() {
         </div>
         {/* Tabs for larger screens */}
         <div className="hidden md:flex flex-row justify-center space-x-4 overflow-x-auto">
-          {stopOptions.map((stopName) => (
-            <button key={stopName} className={`px-4 py-2 text-xl font-semibold border-b-2 
+          {stopOptions.map((stopName, index) => (
+            <button key={index} className={`px-4 py-2 text-xl font-semibold border-b-2 
```

Comment:
we cant use the stopName as the key like it was already?

https://stackoverflow.com/a/46735689

Reply:
yeah, I was getting an error in the console but I think this wasn't the issue

resolved